src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,131 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    /**
+     * File that contains user task information.
+     */
+    private File toRead;
+    /**
+     * Path that contains the file.
+     */
+    private String path;
+
+    public Storage (String path) {
+        this.path = path;
+        toRead = new File(path);
+
+        if (!toRead.exists()) {
+            try {
+                File directory = toRead.getParentFile();
+                if (!directory.exists()) {
+                    directory.mkdirs();
+                }
+                toRead.createNewFile();
+            } catch (IOException e) {
+                System.out.println(e);
+            }
+        }
+    }
+
+    /**
+     * Reads the file and loads the information into TaskList.
+     * If file does not exist in specified location, creates the file.
+     * @return List of the tasks in the file.
+     */
+    public ArrayList<Task> load() {
+        ArrayList<Task> tasks = new ArrayList<>();
+
+        try {
+            Scanner fileReader = new Scanner(toRead);
+            while (fileReader.hasNextLine()) {
+                String line = fileReader.nextLine();
+                String[] info = line.split(",");
+                String taskType = info[0];
+                Task toAdd;
+                switch (taskType) {
+                case "T":
+                    toAdd = new Todo(info[2]);
+                    if (Integer.parseInt(info[1]) == 1) {
+                        toAdd.setComplete();
+                    }
+                    tasks.add(toAdd);
+                    break;
+                case "D":
+                    toAdd = new Deadline(info[2]);
+                    String[] dateArr = info[3].split(" ");
+                    toAdd.setDate(LocalDate.parse(dateArr[0]));
+                    if(dateArr.length > 1) {
+                        toAdd.setTime(dateArr[1]);
+                    }
+                    if (Integer.parseInt(info[1]) == 1) {
+                        toAdd.setComplete();
+                    }
+                    tasks.add(toAdd);
+                    break;
+                case "E":
+                    toAdd = new Event(info[2]);
+                    String[] dateArrEvent = info[3].split(" ");
+                    toAdd.setDate(LocalDate.parse(dateArrEvent[0]));
+                    if(dateArrEvent.length > 1) {
+                        toAdd.setTime(dateArrEvent[1]);
+                    }
+                    if (Integer.parseInt(info[1]) == 1) {
+                        toAdd.setComplete();
+                    }
+                    tasks.add(toAdd);
+                    break;
+                default:
+
+                }
```

Comment:
An empty default case feels a bit redundant here. Would adding a comment to explain why there are no other cases/this default case be reached or simply to remove this be better?